src/widgets/NavigationBar/ui/Notifications.tsx

```
@@ -65,73 +67,116 @@ export default function Notifications() {
-                <div className="w-full flex p-4 gap-4 items-center">
-                  <div className="justify-center items-center flex-shrink-0 flex">
-                    <div
-                      className={`w-2 h-2 rounded-full mt-2 ${
-                        notification.isRead ? 'bg-gray-500' : 'bg-primary'
-                      }`}
+      {open && (
+        <>
+          {/* 배경 오버레이 */}
+          <div className="fixed inset-0 bg-black/20 z-40" onClick={() => setOpen(false)} />
+
+          {/* 모달 드롭다운 */}
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)